### DISCUSSION TITLE
Do we need to support core mapping identifiers?

### DISCUSSION THREAD

**Discussion by matentzn** [mapping-commons/sssom#360] [General]
In a SSSOM TSV, and individual row corresponds to  a metadata record about a mapping, not a core mapping.

The core mapping is comprised of:

1. subject_id
2. predicate_id
3. object_id
4. predicate_modifier

To be able to identify a core mapping in a semantic web context and talk about it, people are asking how the URI representing those mappings should be captured.

Here we need to discuss the why: What is your use case for needing to talk about the core mapping  in isolation of its metadata / justifications?

**Comment by joeflack4** [comment]
FYI I'm partial to the term "mapping quad" for this.
We've been discussing the merits and demerits of this in #359, but for the record I'm ambivalent / opposed unless a strong use case emerges or users start asking for it.

**Comment by nichtich** [comment]
We discussed at TS4NFDI workshop yesterday and agreed to define an identifier for "mapping sameness" based on subject URIs, predicate, object URIs, and whether the mapping is negative. The specification should not depend on a specific format such as JSKOS or SSSOM.

**Reply by nichtich** [reply]
See <https://github.com/ts4nfdi/mapping-sameness-identifier/blob/main/index.qmd> = <https://ts4nfdi.github.io/mapping-sameness-identifier/> for a start.
